apps/dashboard/components/logs/queries/queries-made-by.tsx

```diff
@@ -0,0 +1,31 @@
+import { Avatar, AvatarImage } from "@/components/ui/avatar";
+import { CircleHalfDottedClock } from "@unkey/icons";
+
+type QueriesMadeByProps = {
+  userName?: string;
+  userImageSrc: string;
+  createdString: string;
+};
+
+export const QueriesMadeBy = ({ userName, userImageSrc, createdString }: QueriesMadeByProps) => {
```

Comment:
this works, it's just confusing to read cause you do the `userName` check three times.

it would be easier to understand if we do a single check